src/lib/builders/slider/create.ts

```diff
@@ -55,6 +55,7 @@ export const createSlider = (props?: CreateSliderProps) => {
 		returned: ([$disabled, $orientation]) => {
 			return {
 				disabled: disabledAttr($disabled),
+				'aria-disabled': $disabled ? 'true' : 'false',
```

Comment:
I think we can use the `disabledAttr` helper. It's easier for styling, and according to MDN it's valid behaviour to remove it when not disabled. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled#associated_roles

Reply:
The problem with the helper is that it sets it to the boolean true, not the string, so Tailwind’s `aria-disabled:` doesn’t work. So, you’d need to change the helper or just make another helper for aria disabled.

Reply:
I see what you mean. In that case, `'aria-disabled': $disabled ? 'true' : undefined` should suffice I believe, right?

Reply:
Yeah sure. That should support both use cases `[aria-disabled]` and `[aria-disabled="true"]`.

Reply:
Perfect then :)

Reply:
@abdel-17 it seems Svelte resolves `true` to `"true"` in this case

Reply:
Strange. Looks like `aria-disabled` and `aria-disabled={true}` behave differently when it comes to DOM elements.